examples/arm/aot_arm_compiler.py

```diff
@@ -195,24 +209,33 @@ def forward(self, x):
         ),
     )
     logging.debug(f"Exported graph:\n{edge.exported_program().graph}")
-
     if args.delegate is True:
         edge = edge.to_backend(
             ArmPartitioner(
                 generate_ethosu_compile_spec(
                     "ethos-u55-128",
-                    permute_memory_to_nhwc=True,
+                    permute_memory_to_nhwc=args.model_name
```

Comment:
confusing casting to bool?

Reply:
This is a rough "guess" for whether the memory needs permuting that happens to work on the supported cases so it's not ideal but I figured that it's better than always permuting which broke running add with --delegate. Thanks for the merge :)